### ISSUE TITLE
Better integration of censorship circumvention tools

### ISSUE THREAD

**Issue by hellais** [ooni/probe#653]
We have a bunch of tests that depend on censorship circumvention tools that are written in go.

It would be great if we could ship them as part of ooniprobe and test them via such statically linked compiled binaries.

For the purpose of this I think a good approach would be that of creating a thin wrapper around the go based censorship circumvention tools (obfs4proxy, lantern, psiphon), without duplicating the go runtime.

Here is some information on how this can be achieved:

https://bugs.torproject.org/13770#comment:1
https://trac.torproject.org/projects/tor/attachment/ticket/13770/mkmulti

Basically you do this:

PROGS="git.torproject.org/pluggable-transports/obfs4.git/obfs4proxy git.torproject.org/pluggable-transports/meek.git/meek-client git.torproject.org/pluggable-transports/meek.git/meek-client-torbrowser github.com/yawning/tor-fw-helper"
go get -d $PROGS
./mkmulti multi $PROGS
cd multi
go build

**Comment by hellais** [comment]
This issue is superseeded by https://github.com/ooni/spec/issues/109.
